### ISSUE TITLE
Paths too long on windows 7

### ISSUE THREAD

**Issue by cb88** [arduino-org/ArduinoStudio#2]
When extracting the zip file on Windows 7 with 7zip I got an error while it was copying from the temp directory to the desktop. When I copied it to the C:\ drive it was fine and appears to be working.

Also, the windows 7 built in zip utilities failed to open the file but 7zip opened it and when I used 7zip to test it it was ok.

**Comment by quasto** [comment]
hi @cb88 
we recommend to follow this guide for Windows installation: http://labs.arduino.org/Arduino+Studio+for+Windows. Using 7zip you shouldn't have problems. 
Let us know if you encountered any others problems unzipping the package.
Regards

**Comment by cb88** [comment]
Well I did use 7zip and I did have the problem I mentioned. 

When extracting to my desktop it said the path was too long. I know this is a problem for kicad on windows as well... however the page you linked me to there mentioned nothing about needing to extract the files to a folder close to the root of the drive.

**Comment by quasto** [comment]
@cb88 yes, the page doesn't say anything about extracting in C:\ folder because, using 7zip, we never encountered the problem. In the following picture you can see that I extracted the zip file into a folder "test" which is into the Desktop. 
![screen shot 2015-09-11 at 13 16 49](https://cloud.githubusercontent.com/assets/6087832/9816378/e217a5b6-589c-11e5-8c98-e85e727ef184.png)
![screen shot 2015-09-11 at 13 17 44](https://cloud.githubusercontent.com/assets/6087832/9816660/b7c8b2e4-589e-11e5-9e5b-a327fc8539d0.png)

 We also tested the unzipping process on Win 7, 8.1 and 10 and it works.
Please help me to help you (and other users), how can I reproduce the problem? :confused: 
Thank you for your help

**Comment by cb88** [comment]
![capture _arduino_studiopng](https://cloud.githubusercontent.com/assets/1026000/9819860/6b181acc-587e-11e5-9ae1-edd78b19f614.PNG)

So the only differences I can think of is that I am on a domain with a user folder path equivalent to: C:\Users\username1.DOMAIN1\Desktop

Also 7zip is installed in it's default location. The error does not occur during but extraction but the file copy from temp to the destination folder.

**Comment by quasto** [comment]
Unfortunately I don't have a Windows machine running on a domain so i can not make this test. But I think the problem might be related with Windows domain :pensive:. I need some time to be able to make this test.

**Comment by nicolettedanagrin** [comment]
Try the Long Path Tool

**Comment by gamicklisafisher** [comment]
You should check out ‘Long Path Tool’ it will help you fix the issues.
I had faced similar issue and this tool helped me to resolve this issue.
